src/main/java/org/trigon/commands/GearRatioCalculationCommand.java

```diff
@@ -1,60 +1,99 @@
-import org.littletonrobotics.junction.networktables.LoggedDashboardBoolean;
 import org.littletonrobotics.junction.networktables.LoggedDashboardNumber;
 
 import java.util.function.DoubleConsumer;
 import java.util.function.DoubleSupplier;
 
-public class GearRatioCalculationCommand extends Command {
+/**
+ * A command that calculates and logs the gear ratio of a subsystem by comparing the positions of a rotor and an encoder.
+ */
+public class GearRatioCalculationCommand extends SequentialCommandGroup {
     private final DoubleSupplier rotorPositionSupplier;
     private final DoubleSupplier encoderPositionSupplier;
     private final DoubleConsumer runGearRatioCalculation;
     private final String subsystemName;
+    private final double backlashAccountabilityTimeSeconds;
 
     private final LoggedDashboardNumber movementVoltage;
-    private final LoggedDashboardBoolean shouldMoveClockwise;
 
     private double startingRotorPosition;
     private double startingEncoderPosition;
     private double gearRatio;
 
-    public GearRatioCalculationCommand(DoubleSupplier rotorPositionSupplier, DoubleSupplier encoderPositionSupplier, DoubleConsumer runGearRatioCalculation, SubsystemBase requirement) {
+    /**
+     * Creates a new GearRatioCalculationCommand.
+     *
+     * @param rotorPositionSupplier             a supplier that returns the current position of the rotor
+     * @param encoderPositionSupplier           a supplier that returns the current position of the encoder
+     * @param runGearRatioCalculation           a consumer that runs the gear ratio calculation with a given voltage
+     * @param backlashAccountabilityTimeSeconds the time to wait before logging the gear ratio in order to account for backlash
+     * @param requirement                       the subsystem that this command requires
+     */
+    public GearRatioCalculationCommand(
+            DoubleSupplier rotorPositionSupplier,
+            DoubleSupplier encoderPositionSupplier,
+            DoubleConsumer runGearRatioCalculation,
+            double backlashAccountabilityTimeSeconds,
+            SubsystemBase requirement) {
         this.rotorPositionSupplier = rotorPositionSupplier;
         this.encoderPositionSupplier = encoderPositionSupplier;
         this.runGearRatioCalculation = runGearRatioCalculation;
         this.subsystemName = requirement.getName();
+        this.backlashAccountabilityTimeSeconds = backlashAccountabilityTimeSeconds;
 
         this.movementVoltage = new LoggedDashboardNumber("GearRatioCalculation/" + this.subsystemName + "/Voltage", 1);
-        this.shouldMoveClockwise = new LoggedDashboardBoolean("GearRatioCalculation/" + this.subsystemName + "/ShouldMoveClockwise", false);
 
         addRequirements(requirement);
+        addCommands(
+                getGearRatioCalculationCommand().alongWith(getBacklashAccountabilityCommand()),
+                getCalculateGearRatioCommand()
+        );
```

Comment:
using a sequential command just seems weird and confusing. moreover, the `getGearRatioCalculationCommand` doesn't ever end, to the second command won't be called. you want to print the value onEnd, which makes this more suitable for a normal functional command.